backend/routes.js

```diff
@@ -257,8 +266,40 @@ export default class Servers {
                         existingUsers.push({username, accessToken: access_token});
                         status = 201;
                     }
-                    saveSetting('USERS', existingUsers);
-                    res.json(status, {});
+
+                    if (contains(username, getSetting('ALLOWED_USERS'))) {
+                        res.setCookie('plotly-auth-token', access_token, {'path': getSetting('DB_CONNECTOR_URL')});
+
+                        const db_connector_access_token = generateAndSaveAccessToken();
+                        res.setCookie('db-connector-auth-token',
+                                      db_connector_access_token, {
+                                          'maxAge': getAccessTokenExpiry(),
+                                          'path': getSetting('DB_CONNECTOR_URL'),
+                                          'secure': getSetting('PLOTLY_API_SSL_ENABLED')
+                                      });
+                        res.setCookie('db-connector-user', username, {'path': '/'});
```

Comment:
let's make these cookie options a `CONSTANT` as in https://github.com/plotly/plotly-database-connector/pull/180/files/e7034e717d8602fa95158af34447b5a8d4e7f712..7204ec66554f53e5732be3d1da64fcd020c0323d#r143544790 so that we can be sure that they're consistent everywhere